dp/tools/deployment/vendor/elasticsearch.yml

```diff
@@ -24,6 +24,12 @@ spec:
           env:
             - name: discovery.type
               value: single-node
+          readinessProbe:
```

Comment:
Let's maybe investigate flake first before proposing this change, I'm not sure if it helps

Reply:
While it was not proven explicitly (we did not notice the integ tests failing) we did see an indication that this fix removes the flakiness. We saw that fluentd-forward did have problems connecting to ES, and entered a long backoff connection retry.
The test managed to succeed, but we did see backoff timeouts kicking in on fluentd-forward due to ES unavailability.